src/main/java/edu/nextstep/camp/carracing/view/ResultView.java

```diff
@@ -0,0 +1,28 @@
+package edu.nextstep.camp.carracing.view;
+
+import edu.nextstep.camp.carracing.domain.Car;
+import edu.nextstep.camp.carracing.domain.Cars;
+
+public class ResultView {
+    private static final String RESULT_MESSAGE = "실행 결과";
+
+    private ResultView() {
+        throw new IllegalStateException("인스턴스 생성이 불가능한 클래스입니다.");
+    }
+
+    public static void printResultMessage() {
+        System.out.println('\n' + RESULT_MESSAGE);
```

Comment:
```suggestion
        System.out.println(System.lineSeparator() + RESULT_MESSAGE);
```
OS 환경에 따라 개행 문자가 다르므로, 하드코딩된 `\n` 대신 
`lineSeparator()`를 사용하는 것이 좋겠습니다!

Reply:
감사합니다. `System.lineSeperator()`를 사용하는 방식으로 변경했습니다. 몰랐던 사실이네요 ☺️